behandlingsprosess/src/main/java/no/nav/foreldrepenger/behandling/steg/beregnytelse/Etterbetalingtjeneste.java

```diff
@@ -32,11 +32,14 @@ private Etterbetalingtjeneste() {
      */
     public static EtterbetalingskontrollResultat finnSumSomVilBliEtterbetalt(LocalDate dagensDato, BeregningsresultatEntitet forrigeRes, BeregningsresultatEntitet nyttBeregningsresultat) {
         var sisteDagSomErUtbetalt = finnSisteUtbetalteDato(dagensDato);
+
         var originalUtbetaltTidslinje = lagUtbetaltTidslinje(sisteDagSomErUtbetalt, forrigeRes);
         var nyUtbetaltTidslinje = lagUtbetaltTidslinje(sisteDagSomErUtbetalt, nyttBeregningsresultat);
 
+        var overlappendeNyTidslinje = nyUtbetaltTidslinje.intersection(originalUtbetaltTidslinje);
```

Comment:
Foreslår at vi legger ut logging av denne aller først.
Slik at det logger resultat as is og den nye overlappssjekken.
Fint om det kan logge et par dager - så kan jeg bli med å finjustere denne tjenesten (tenke om vi skal sjekke diff på andelsnivå)

Reply:
Gjerne med diff for ny sjekk på 30000 for loggformål - det er en gjennomsnittlig månedsrefusjon

Reply:
Vi har allerede lagt inn kun logg, oppretter ikke noe aksjonspunkt enda :) https://github.com/navikt/fp-sak/pull/6083/files#diff-469ac884ee96df7aacc0b4d10e2c6e9f44cd310efa78e1bab5822a0557a5abcfR130

Reply:
Vi har endret til 60 000 kr, vi ble enige om at det var et greit sted å starte.